src/viam/components/component_base.py

```diff
@@ -45,7 +45,7 @@ async def do_command(self, command: Mapping[str, ValueTypes], *, timeout: Option
 
     async def get_geometries(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None) -> List[Geometry]:
         """
-        Get all geometries associated with the Component, in their current configuration, in the frame of the Component.
+        Get all geometries associated with the Component, in their current configuration, in the `frame <https://docs.viam.com/services/frame-system/>`__ of the Component.
```

Comment:
Yay links!
[nit] IDK why component is capitalized.
```suggestion
        Get all geometries associated with the component, in their current configuration, in the `frame <https://docs.viam.com/services/frame-system/>`__ of the component.
```